packages/jellyfish-api-core/__tests__/category/governance/createVoc.test.ts

```diff
@@ -0,0 +1,101 @@
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { ProposalStatus, ProposalType } from '../../../src/category/governance'
+import { RpcApiError } from '@defichain/jellyfish-api-core'
+import { GovernanceMasterNodeRegTestContainer } from './governance_container'
+
+describe('Governance', () => {
+  const container = new GovernanceMasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should createVoc', async () => {
+    const proposalTx = await client.governance.createVoc('new vote of confidence')
+    await container.generate(1)
+
+    expect(typeof proposalTx).toStrictEqual('string')
+
```

Comment:
You can see other files in the project. 
For txid, we check string and length together. For eg,

<img width="599" alt="Screenshot 2021-07-27 at 11 25 37 PM" src="https://user-images.githubusercontent.com/7101806/127181878-866f4c37-2626-4b40-9e82-6493c7432256.png">


Hence, you can add `expect(proposalTx.length).toStrictEqual(64)`